app/models/manageiq/providers/storage_manager/cinder_storage_manager.rb

```diff
@@ -0,0 +1,86 @@
+#
+#
+# TODO (hsong) Storage Manager
+#
+#
+
+
+class ManageIQ::Providers::StorageManager::CinderStorageManager < ManageIQ::Providers::StorageManager 
+  require_nested :EventCatcher
+  require_nested :EventParser
+  require_nested :RefreshParser
+  require_nested :RefreshWorker
+  require_nested :Refresher
+
+  has_many :cloud_volumes,                 :foreign_key => :ems_id, :dependent => :destroy
+  has_many :cloud_volume_snapshots,        :foreign_key => :ems_id, :dependent => :destroy
+  has_many :cloud_volume_backups,          :foreign_key => :ems_id, :dependent => :destroy
+
+  # Auth and endpoints delegations, editing of this type of manager must be disabled
+  delegate :authentication_check,
+           :authentication_status,
+           :authentication_status_ok?,
+           :authentications,
+           :authentication_for_summary,
+           :zone,
+           :connect,
+           :verify_credentials,
+           :with_provider_connection,
+           :address,
+           :ip_address,
+           :hostname,
+           :default_endpoint,
+           :endpoints,
+           :to        => :parent_manager,
+           :allow_nil => true
+
+  supports :cinder_storage do
+    if self.parent_manager
+      unsupported_reason_add(:cinder_storage, self.parent_manager.unsupported_reason(:cinder_storage)) unless 
+        self.parent_manager.supports_cinder_serive?
```

Comment:
Yes, added it in the parent manager, e.g openstack provider. Thanks for the reminder.